src/App.js

```diff
@@ -1,13 +1,25 @@
 import { data } from "./db.js";
```

Comment:
I don't think we need this line anymore, seeing as we are creating a new state variable called `data`.